react/src/lib/components/DeckGLMap/components/Map.tsx

```diff
@@ -589,11 +600,24 @@ const Map: React.FC<MapProps> = ({
                     scaleUnit={coordinateUnit}
                 />
             ) : null}
-
             <StatusIndicator layers={deckGLLayers} isLoaded={isLoaded} />
-
             {coords?.visible ? <InfoCard pickInfos={hoverInfo} /> : null}
-
+            {Object.keys(getCameraPosition).length !== 0 ? (
```

Comment:
We can have this div component in storybook, if it's here this div will be visible on the consuming component side too.